dataflow/src/ops/grouped/mod.rs

```diff
@@ -222,6 +222,7 @@ where
                         }
                     };
 
+                    let old = rs.get(0);
```

Comment:
I feel like we can probably take advantage of `old` being `Cow::Owned` here too, but probably not worth spending time on. Maybe add a `TODO`?

Reply:
Yeah, agreed. Tried briefly but got shot down by the borrow checker, so added a TODO for now.